### PR TITLE
CM-239 Tablas de salud (illness, disease y note)

### PR DESCRIPTION
**Pull request by merlindk**

### CODE REVIEW

File: health-be/disease.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+create table disease
+(
+	id serial not null
+		constraint batch_pkey
+			primary key,
+	name varchar(50) not null,
+	establishment_cuig varchar(50) not null,
+	description varchar(500),
+	duration bigint default 0
```
Comment (FranciscodeMaussion) [inline health-be/disease.sql:9]: in days?

File: health-be/illness.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+\connect health
+*/
+CREATE EXTENSION pgcrypto;
+create table illness
+(
+	id uuid default gen_random_uuid() not null
+		constraint illness_pkey
+			primary key,
+	starting_date timestamp default now(),
+	ending_date timestamp default now(),
```
Comment (FranciscodeMaussion) [inline health-be/illness.sql]: maybe we should default this to null..

File: health-be/note.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+create table note
```
Comment (FranciscodeMaussion) [inline health-be/note.sql:1]: to give an order to all notes, we can add a date or a note number

**Review by FranciscodeMaussion — APPROVED** [review]

File: health-be/note.sql (hunk 3 above)
Comment (merlindk) [reply health-be/note.sql:1]: added

**Pull request merged**